HPXMLtoOpenStudio/resources/electric_panel.rb

```diff
@@ -0,0 +1,251 @@
+# frozen_string_literal: true
+
+# Collection of methods related to electric panel load calculations.
+module ElectricPanel
+  # Calculates load-based capacity and breaker spaces for an electric panel.
+  #
+  # @param hpxml_header [HPXML::Header] HPXML Header object (one per HPXML file)
+  # @param hpxml_bldg [HPXML::Building] HPXML Building object representing an individual dwelling unit
+  # @param electric_panel [HPXML::ElectricPanel] Object that defines a single electric panel
+  # @return [nil]
+  def self.calculate(hpxml_header, hpxml_bldg, electric_panel)
+    capacity_types = []
+    capacity_total_watts = []
+    capacity_total_amps = []
+    capacity_headroom_amps = []
```

Comment:
What's the point of these local variables that will simply get assigned to the electric panel? Just use `electric_panel.capacity_types = []` (and further down `electric_panel.capacity_types << ...`).